lblod/frontend-gelinkt-notuleren
# GN-4977: add meeting details modal to inauguration meeting creation flow

### Overview
This PR adds a meeting details modal to the inauguration meeting creation flow to allow users to specify the location and time of the meeting.
Differences compared to common meeting creation modal:
- Administrative unit is not editable
- Modal title is slightly different

Additionally, this PR also includes a modification in how the governing body (bestuurseenheid) is selected for the inauguration meeting:
- The `binding-einde` should be undefined or gte than the current date. (is this correct?)

##### connected issues and PRs:
[GN-4977](https://binnenland.atlassian.net/browse/GN-4977)

### How to test/reproduce
- Start the app
- Visit the meeting inbox
- Test if both the common meeting and inauguration meeting creation flows works as expected

### Checks PR readiness
